custom-code-library.hbs

```diff
@@ -64,7 +64,7 @@
             <div>
                 {{#get "tags" limit="all"}}
                     {{#foreach tags visibility="internal"}}
-                        <h1>{{{trimString name}}} Routines</h1>
+                        <h1 class="code-library-header">{{ name }} Routines</h1>
```

Comment:
Seems to have no effect. I've tried placing the script in a few different places, too. I replaced `{{ name }}` with just plain text: "#clsApp Routines" and nothing had been altered in my dev environment.

Reply:
oh, yeah the code pen I tested it in tricked me...